test/framework/resource/aws/ec2/manager.go

```diff
@@ -17,8 +17,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/aws/amazon-vpc-resource-controller-k8s/pkg/aws/vpc"
```

Comment:
What is this import for? I am not seeing other changes using this package in this commit?

Reply:
Hi hao that import was added in PR https://github.com/aws/amazon-vpc-resource-controller-k8s/pull/418. I didn't cherry pick this PR as this caused regression. But recent code change does use this import so added it.

Reply:
Got it. Thanks